opengemini/client.go

```diff
@@ -1 +1,71 @@
 package opengemini
+
+import (
+	"crypto/tls"
+	"errors"
+)
+
+type Options = func(client *Client) error
```

Comment:
what's Options used for?

Reply:
sorry I forgot to delete it , I'll modify it later thanx ：）